package.json

```diff
@@ -21,7 +21,7 @@
     "start": "set NODE_ENV=development && ts-node src/server/index.ts --colors --profile",
     "start-prod": "set NODE_ENV=production && ts-node src/server/index.ts --progress --colors --profile",
     "test": "run-s test:lint test:types test:unit",
-    "test:lint": "node_modules/eslint/bin/eslint.js src test && node_modules/tslint/bin/tslint src/**/*.ts",
+    "test:lint": "eslint src test && tslint src/**/*.ts",
```

Comment:
this should probably be `tslint src/**/*.tsx?`

Reply:
Shouldn't it be both?

Reply:
`src/**/*.ts*` I guess?